app/assets/stylesheets/components/_error-message.scss

```diff
@@ -2,12 +2,8 @@
 
 .app-c-error-message {
   display: block;
-
   margin-bottom: $app-spacing-scale-3;
-
   clear: both;
-
   color: $app-error-colour;
```

Comment:
Can we use `$govuk-error-colour`? It looks like `$app-error-colour` is defined [here](https://github.com/alphagov/government-frontend/blob/master/app/assets/stylesheets/components/helpers/_variables.scss#L24) as being equal to `$red`, which I'm going to guess is a colour from toolkit.

Reply:
Done, good spot, thanks.